packages/php-db-import-export/src/Backend/Bigquery/ToStage/FromGCSCopyIntoAdapter.php

```diff
@@ -70,6 +71,14 @@ public function runCopyCommand(
             ->quote($source->getCsvOptions()->getEnclosure())
             ->allowQuotedNewlines(true);
 
+        if ($importOptions->importAsNull() !== []) {
+            // BigQuery allows only one null marker
+            // we implicitly use the first one and ignore others if any
+            $loadConfig->nullMarker(
+                $importOptions->importAsNull()[0],
+            );
+        }
```

Comment:
docela zajímavá věc nepodařilo sa mě ten null marker nijak zrušit. Default je to empty string. Nejde nastavit na nic, při nastavení na null dělá default, když sa nepošle tak dělá default :)